test/Twig/Tests/TemplateTest.php

```diff
@@ -255,15 +259,17 @@ public function testGetAttributeStrict($defined, $value, $object, $item, $argume
         if ($defined) {
             $this->assertEquals($value, $template->getAttribute($object, $item, $arguments, $type));
         } else {
-            try {
-                $this->assertEquals($value, $template->getAttribute($object, $item, $arguments, $type));
-
-                throw new Exception('Expected Twig_Error_Runtime exception.');
-            } catch (Twig_Error_Runtime $e) {
-                if (null !== $exceptionMessage) {
-                    $this->assertSame($exceptionMessage, $e->getMessage());
+            if (!$defined) {
```

Comment:
this ``if`` will always be true, as it is inside the else clause of the inverse condition